trustme/__init__.py

```diff
@@ -419,6 +420,36 @@ def configure_trust(self, ctx):
                 "unrecognized context type {!r}"
                 .format(ctx.__class__.__name__))
 
+    @classmethod
+    def from_pem(cls, cert_path, private_key_path):
```

Comment:
Taking paths limits this to certs that are stored in files on disk, which is kind of an arbitrary limitation. Maybe it should accept the actual PEM data instead? (Or file objects, but that feels unnecessarily complicated when we're talking about data that's a few kilobytes at most.)

Reply:
Sure. This is up to you! I can accept `trustme.Blob`s, or bytes, or file-like objects

Reply:
(My reasoning was that if you use `from_pem` then you're likely to have the files on disk, but well, this kind of thinking led to only accept certificates on disks in many APIs, so it's flawed indeed)

Reply:
I've implemented the bytes option. It certainly simplifies the code! Please take another look.